run.sh

```diff
@@ -618,14 +618,18 @@ acraengdemo_launch_project_timescaledb() {
 
 acraengdemo_launch_project_python-searchable() {
     COSSACKLABS_ACRA_VCS_URL=${COSSACKLABS_ACRA_VCS_URL:-'https://github.com/cossacklabs/acra'}
-    COSSACKLABS_ACRA_VCS_BRANCH=${COSSACKLABS_ACRA_VCS_BRANCH:-0.93.0}
+    # using commit instead of version/tag to use the commit with specific SqlAlchemy from examples/python
+    # https://github.com/cossacklabs/acra/pull/669/commits/be2ab1a4440e105dee1423ae21da1ee74e842801
+    # and some examples/python fixes
+    # https://github.com/cossacklabs/acra/commit/e1773020a881a39d126724bd3992b9b436173fa0
+    COSSACKLABS_ACRA_VCS_REF=${COSSACKLABS_ACRA_VCS_REF:-'e1773020a881a39d126724bd3992b9b436173fa0'}
     if [ -d "${PROJECT_DIR}/acra" ]
     then
-      git -C "${PROJECT_DIR}/acra" checkout "$COSSACKLABS_ACRA_VCS_BRANCH";
+      git -C "${PROJECT_DIR}/acra" checkout "$COSSACKLABS_ACRA_VCS_REF";
```

Comment:
If I understand correctly, this commit is pinned because it was done after the 0.95.0 and we don't have a tag to refer to it, right?

Reply:
Ok, thanks! Could you then add a tag `TODO` or something to mark this as a temporal workaround?

Reply:
> 0.95.0

actually we have release/0.96.0 )) we forgot some releases to create separate branches, but in general have)